### ISSUE TITLE
.storage after property name 

### ISSUE THREAD

**Issue by kimdv** [evermeer/EVCloudKitDao#66]
Hello! 

When i use .connect method i get this error: 

```
WARNING: The class 'Aircrafts' is not key value coding-compliant for the key 'ACType.storage'
 There is no support for optional type, array of optionals or enum properties.
As a workaround you can implement the function 'setValue forUndefinedKey' for this. See the unit tests for more information
```

Maybe its just me being a little stupid or misunderstood something! :)

// Kim

**Comment by evermeer** [comment]
In CloudKit, did you already have a field named "ACType.storage" ?
A property name can not have a dot.
In your Aircrafts object, what property should this go to? 
Is the property called ACType? or storage?

If the existing data contains that name, then you could create a workaround in your Aircrafts object by implementing the function 'setvalue forUndefinedKey' Here is some sample code for that:
https://github.com/evermeer/EVReflection/blob/ddb5bc7c754a388745457610a5b087b9ba4d4fec/EVReflection/EVReflectionTests/EVReflectionConversionOptionsTest.swift#L85-L85

Of course it would be better to just fix the data.

**Comment by kimdv** [comment]
I have a property named "ACType"

**Comment by evermeer** [comment]
Is ACType an object which has a property named storage?

**Comment by kimdv** [comment]
No, ACType is an String

**Comment by evermeer** [comment]
Ah... The lazy is the problem. You do have to be aware that lazy properties won't be initialized unless they are accessed. Using reflection does not actually call the properties. It will ask it current state. Because of this they will have a default value of nil. For some reason Swift has added the .storage to the property name.  I will try to see if I can fix this.

**Comment by evermeer** [comment]
I was able to fix this in EVReflection version 2.44.0 (please do a pod update)
If you do a pod update it should work.
It could be that existing records still have the .storage in the field name.
It would be best to remove these records.
